src/components/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { InputWrapper, Label, StyledIcon, StyledInput } from './Input.styled';
+import { IProps } from './Input.model';
+
+const Input = ({ type = 'text', icon: Icon, label, ...props }: IProps): JSX.Element => {
```

Comment:
Did you check what is the behaviour if I try to use type number/date or even something different?
Each component should have props `className` to show extended components in proper way

Reply:
![Bez tytułu](https://user-images.githubusercontent.com/29055880/92414275-d9fc6d80-f153-11ea-800c-7bb2cce741d9.png)